tests/datetime.spec.js

```diff
@@ -813,6 +814,34 @@ describe('Datetime', () => {
 				});
 			});
 
+			it('displayTimeZone -> value should change format (undefined->America/New_York)', () => {
+				const date = new Date(2000, 0, 15, 2, 2, 2, 2),
+					momentDate = moment(date),
+					component = utils.createDatetime({ value: momentDate }),
+					displayTimeZone = (moment.tz.guess() === 'America/New_York' ? 'America/Los_Angeles' : 'America/New_York');
+
+				const valueBefore = utils.getInputValue(component);
+				component.setProps({ displayTimeZone: displayTimeZone }, () => {
```

Comment:
I don't trust assertions inside these callbacks, they don't seem to work sometime. Did you verify with something like `expect(true).toEqual(false)`?